ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/Config.scala

```diff
@@ -42,6 +45,16 @@ object Config {
         .text(
           "Sorts the computed write set. Older exports sorted before writing. Newer versions order them intentionally.")
         .action((_, c) => c.copy(sortWriteSet = true))
+      opt[Unit]("index-only")
+        .text(
+          "Run only the indexing step of the integrity checker (useful tp benchmark the indexer).")
+        .action((_, c) => c.copy(indexOnly = true))
+      opt[String]("jdbc-url")
+        .text("External JDBC url (useful for running against PostgreSQL).")
+        .action((jdbcUrl, c) => c.copy(jdbcUrl = Some(jdbcUrl)))
+      opt[Unit]("full-metrics")
```

Comment:
Let's name it `print-metrics`/`report-metrics` or similar then. I.e., `full-metrics` doesn't imply that we'll see these metrics at the end.

Reply:
Done.